webapp/packages/plugin-too-many-sessions/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@cloudbeaver/plugin-too-many-sessions",
```

Comment:
please remove completely and move to plugin-sso as sso.error.html or something like this